ROADMAP.md

```diff
@@ -3,53 +3,61 @@
 This document is meant to provide high-level but actionable objectives for future kube-aws deveploment.
 Please file an issue to make suggestions on this roadmap!
 
-## Every stage
+## Every release
 
   * Provide article walking users through:
     * Setting up a cluster from scratch
     * Using/enabling new features
     * (Breaking changes)
   * Drop deprecated configuration syntax and flags, options
+  * Revise this roadmap
 
-## Stage 1: v0.9.2
+## v0.9.2
 
   * Node Pools
     * Worker nodes optionally powered by Spot Fleet
   * Clean cluster upgrades (preventing downtime, make sure they succeed)
 
-## Stage 2: v0.9.3
+## v0.9.3
 
   * Kubernetes 1.5.1
      * Auto-scaled kube-dns
   * Self-hosted Calico
   * Very limited, almost theoretical support for automatic reconfiguration of cluster-autoscaler
 
-## Stage 3: v0.9.4
+## v0.9.4
 
-  * Cluster Auto Scaling
-    * Support for auto-scaling worker nodes with dynamic reconfiguration of cluster-autoscaler
+  * Kubernetes 1.5.3
   * Work-around the 16KB userdata limit in size
   * Experimental support for
     * Private subnets and NAT gateways for etcd, controller and worker nodes
     * Deployments to existing subnets
-
-## Stage 4: v0.9.5
-
   * Rethink how node pools are implemented
     * See https://github.com/coreos/kube-aws/issues/238
-  * Internal domain and custom hostnames support for etcd nodes?
+
+## v0.9.5
+
   * etcd improvements
-    * Backups
-    * Recovery
+    * Automatic recovery from temporary node failures
+    * Manual disaster recovery w/ or w/o recreating etcd nodes
+  * Experimental support for an internal domain and custom hostnames for etcd nodes
+    
+## v0.9.6
+
+  * Cluster Auto Scaling
+    * Support for auto-scaling worker nodes via:
+      * Dynamic reconfiguration of cluster-autoscaler
+      * Automatic discovery of target node pools for cluster-autoscaler
```

Comment:
Probably we need https://github.com/kubernetes/contrib/pull/1982#discussion_r102573186 to be merged on CA side or otherwise we have to implement a companion service periodically updates CA's configmap with ASGs on which the autoscaling enabled.